prompt_engineering/prompt_engineering.py

```diff
@@ -70,6 +69,49 @@ def create_text_from_questions_table():
     return questions_output[:-1]
```

Comment:
regarding create_text_from_questions_table(), you don't use it so might as well delete it.
If you don't delete it please consult with me cause I think we should rethink it

Reply:
deleted